README.md

```diff
@@ -1 +1,69 @@
-# budget-app
+# Budget app
+
+> The Budget app is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+![UML class diagram](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/images/erd_diagram.png)
+
+## Built With
+
+- _**Ruby**_
+- _**Rails**_
+- _**PostgreSQL**_
+
+### Preview
+
+![This is gif of our project](screenshot.gif)
+
+## Live Demo
+
+[Live Demo Link](https://thawing-sierra-32352.herokuapp.com/)
+
```

Comment:
- [ ] Kindly make sure you follow the requirement for the project documentation. You need to record a video with :
- a description of the project.
- a demo of the project features.
- you should also highlight some interesting piece of code or something you built that you are very proud of.

![image](https://user-images.githubusercontent.com/65068771/170599539-d37f6af9-b0fa-40fa-9636-84dafeb51ebd.png)

Reply:
Forgot to add it 🙈